openwisp_monitoring/monitoring/migrations/0005_create_ping.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+from openwisp_monitoring.check.tasks import auto_create_ping
+
+from openwisp_controller.config.models import Device
+
+
+def create_device_ping(apps, schema_editor):
+    for device in Device.objects.all():
+        auto_create_ping(model=Device.__name__.lower(),
+                         app_label=Device._meta.app_label,
+                         object_id=str(device.pk),
+                         created=True
+        )
```

Comment:
indentation here looks odd, can you move this parenthesis at the end of the previous line?